Shared/ForChildren/ForChildrenView.swift

```diff
@@ -8,13 +8,49 @@
 import SwiftUI
 
 struct ForChildrenView: View {
+    @EnvironmentObject var dataStore: ForKidsDataStore
+    
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack {
+            if dataStore.forKids != nil {
+                ScrollViewReader { proxy in
```

Comment:
Tady to proxy nepotřebuješ :) Take stačí jenom to ScrollView

Proxy potřebujeme pro rychlou navigaci aby jsme mohli měnit programově scroll

Reply:
Fixed